sql/20190424173430_create_clients_table.sql

```diff
@@ -0,0 +1,14 @@
+-- migrate:up
+CREATE TABLE "clients" (
+    "id"                SERIAL  NOT NULL,
+    "first_name"        TEXT     NOT NULL,
```

Comment:
better way is to use varchar(LENGTH) for determining max length

Reply:
I open my practice work for Data Base & Network Technologies and see there TEXT fields everywhere, actually German Krimmel told us to use TEXT fields.